### DISCUSSION TITLE
Why historical data of ASA price changes are not implemented by ASA Stats?

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#42] [General]
A bit of conversation emerged in [our issues](https://github.com/asastats/channel/issues/40), so I'm continuing with it here:

- - - - -

Why historical data of ASA price changes are not implemented by ASA Stats?

The goal for ASA Stats in pre-DAO phase is to create a solid ground for the DAO from the business perspective. We currently have no plans to implement historical data of ASA price changes directly on ASA Stats. However, ASA Stats will be encouraging partnerships and sponsoring developers once our API is ready to create advanced web widgets for use in the portfolio view (e.g. ASA Price change Widget). We have no intention to execute trades on our platform.

The final version should be Added to "Service Users" section in FAQ [Page](https://www.asastats.com/faq/).

- - - - -

Can you please involve more of the reasoning from the original comment here as that's the real reason...

There are no hidden thoughts in ASA Stats and the real and pretty much the only reason why we don't implement historic data (the question) is because we want to encourage people to use our API, among other reasons, because of that.

We are a business, and we don't want to create obstacles for our business.
- - - - -

**Comment by ipaleka** [comment]
This is a topic that the majority hasn't succeeded in comprehending (btw, @SCN9A I'm sure you do understand, the fact that you created the answer doesn't relate to this post), so I'll try to explain it through an example of what I mean when I say we are a business.

Let's say some mobile app emerges using our API, let's call it "Moist Skin". They are serious players with a lot of money involved in marketing and with an enormous social media presence.

They start with some parts of our API implemented in their API, but eventually they implement every piece of our API. At the same time, only a few enthusiastics continue to use https://www.asastats.com. Their board of directors decides to approach the ASA Stats DAO with a proposition of buying the ASA Stats. The DAO votes Yes, and Moist Skin makes all the open-source repositories read-only and continues to develop code in their own branch as closed source.

How would I personally call this outcome?

**A SUCCESS.**

**Reply by wxivd** [reply]
I think it comes down to what investment model we adopt: **Build to sell vs Build to grow**. Each has pros and cons. 

Selling prioritise profitability, whereas growth is more focused on the Brand. I understand your scenario is hypothetical beyond the DAO stage and to the future. 

I long to see ASA Stats as a brand evolving, building our own products, establishing/addressing market needs and scaling up our operations. We sell services but not the Brand.

My point is that developing advanced features for ASA Stats users could be done by third parties (Selling API to software companies) or in-house by sponsoring developers (Profit from selling advanced features to end user).

**Reply by ipaleka** [reply]
Those are more than valid points, for sure. The thing is that the goal of the Team I represent is to transfer custody over the project to the DAO, respectively to develop and deploy everything from our [roadmap](https://github.com/asastats/docs/blob/main/roadmap.md). If we manage to increase our development funds then the vast majority of those funds will be spent on reaching that goal sooner. 

But that doesn't conflict with your in-house *proposal*. Our user widgets system is designed that way where we will be encouraging developers to use our API and we will be donating funds for the development of the functionalities we find important to have.

**Comment by ipaleka** [comment]
So, the Q/A proposed by @SCN9A goes like this:

- - - - -

**Why historical data of ASA price changes are not implemented by ASA Stats?**

The goal for ASA Stats in pre-DAO phase is to create a solid ground for the DAO from the business perspective. We currently have no plans to implement historical data of ASA price changes directly on ASA Stats. However, ASA Stats will be encouraging partnerships and sponsoring developers once our API is ready to create advanced web widgets for use in the portfolio view (e.g. ASA Price change Widget). We have no intention to execute trades on our platform.

- - - - -

That answer is meaningful and clean, but my opinion is that it doesn't bring exactly the answer to *why*.

"We currently have no plans to implement historical data of ASA price changes directly on ASA Stats." Why we don't have them?

Afaic "However" in "However, ASA STATS will be..." should be somehow changed to "Because", and together with that we should emphasize why's that, sort of because our core business is selling the API calls.

**Reply by wxivd** [reply]
@ipaleka What are your thoughts on this iteration

The goal for ASA Stats in pre-DAO phase is to create a solid ground for the DAO from the business perspective (i.e. ASA Stats API Service). Therefore, we currently have no plans to implement historical data of ASA price changes directly on ASA Stats. However, ASA Stats will be encouraging partnerships and sponsoring developers once our API is ready to create advanced web widgets for use in the portfolio view (e.g. ASA Price change Widget). We have no intention to execute trades on our platform.

**Reply by ipaleka** [reply]
Tbh mate, this discussion is open to getting that answer, but the actual goal is to discuss this "problem". I've spent countless minutes explaining in our Discord the same thoughts and all of that is gone for good as Discord is a pseudo-public space.

And now I've got [a proper link](https://github.com/asastats/channel/discussions/42). :)

**Reply by ipaleka** [reply]
> @ipaleka What are your thoughts on this iteration
> 
> The goal for ASA Stats in pre-DAO phase is to create a solid ground for the DAO from the business perspective (i.e. ASA Stats API Service). Therefore, we currently have no plans to implement historical data of ASA price changes directly on ASA Stats. However, ASA Stats will be encouraging partnerships and sponsoring developers once our API is ready to create advanced web widgets for use in the portfolio view (e.g. ASA price change widget).

Deployed, thanks! 

Tho without the last sentence as it doesn't correlate to the question.
